src/orquestra/sdk/mlflow/_connection_utils.py

```diff
@@ -5,11 +5,128 @@
 """Utilities for communicating with mlflow."""
 
 import os
+import warnings
 from pathlib import Path
+from typing import Optional, Tuple
 
+from requests import Response, Session
+
+from orquestra import sdk
+from orquestra.sdk._base import _env
 from orquestra.sdk._base._services import ORQUESTRA_BASE_PATH
+from orquestra.sdk.schema.configs import ConfigName
 
 DEFAULT_TEMP_ARTIFACTS_DIR: Path = ORQUESTRA_BASE_PATH / "mlflow" / "artifacts"
+RESOURCE_CATALOG_URI: str = "http://orquestra-resource-catalog.resource-catalog"
+
+
+# region: private
+def _is_executing_remoteley() -> bool:
+    """
+    Determine whether the code is being executed locally, or on a cluster/studio.
+    """
+    # TODO: at present this uses the platform-specific environment variables, assuming
```

Comment:
**nitpick**: please link an SDK JIRA ticket so we don't have a hanging TODO

Reply:
I think according to this post:https://zapatista.slack.com/archives/C02DN7BA04E/p1689085958796819 we already have ORQ_IN_CLUSTER, right?